packages/client/hmi-client/src/components/operator/tera-operator-menu.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<section ref="operatorMenu" :style="style">
+		<Button
+			v-if="node"
+			ref="openMenuButton"
+			class="button"
+			type="button"
+			size="small"
+			@click="showMenu"
+			icon="pi pi-plus"
+			:style="{
+				top: `${currentPortPosition.y - 18}px`,
+				left: `${currentPortPosition.x + 40}px`
+			}"
+			aria-haspopup="true"
+			aria-controls="overlay_menu"
+			severity="secondary"
+		/>
+		<Menu ref="menu" id="overlay_menu" :model="menuItems" :popup="true" />
+	</section>
+</template>
+<script setup lang="ts">
+import { ref } from 'vue';
+import type { WorkflowNode } from '@/types/workflow';
+import Menu from 'primevue/menu';
+import Button from 'primevue/button';
+
+defineProps<{
+	node: WorkflowNode<any> | null;
+	style: { top: string; left: string };
+	currentPortPosition: any;
```

Comment:
I'm wondering if you should leave this to the parent on how to display the component.
Try using relative absolute positioning with the parent instead.